src/data/committee.js

```diff
@@ -18,17 +18,17 @@ const committee = {
           image: 'serene_lim.jpg',
           course: 'BASc Arts and Sciences (3rd year)',
           languages: 'English... kidding a bit of Python',

```

Comment:
I think we want to get rid of `languages` completely, so please remove them wherever someone specified them previously. We will just stick with `ai_concept`.